packages/patternfly-4/react-core/src/components/AboutModal/AboutModalBoxHero.js

```diff
@@ -14,9 +16,14 @@ const defaultProps = {
   className: ''
 };
 
-const AboutModalBoxHero = ({ className, ...props }) => (
-  <div {...props} className={css(styles.aboutModalBoxHero, className)} />
-);
+const AboutModalBoxHero = ({ className, bgImgSrc, ...props }) => {
+  const bgStyle = (bgImgSrc !== '') ? {
+    "--pf-c-about-modal-box__hero--sm--BackgroundImage": `url(${bgImgSrc})`
+  } : {};
+  return (
+    <div {...props} style={bgStyle} className={css(styles.aboutModalBoxHero, className)} />
```

Comment:
What happens if it doesn't find the image?

Reply:
Good question, so if you're referencing a local module like in the example, and it can't be found, your IDE should notice this and throw an error. If you're providing a value like `https://www.site.com/image.svg` and that asset doesn't properly fetch, you'd get a standard 404 just as if you provide an inaccessible asset in any regular background definition, in which case you'd get a black background.